packages/back-end/src/controllers/admin.ts

```diff
@@ -26,3 +35,104 @@ export async function getOrganizations(
     total,
   });
 }
+export async function getUsersForOrg(
+  req: AuthRequest<unknown, { orgId: string }>,
+  res: Response
+) {
+  if (!req.superAdmin)
+    return res.status(403).json({
+      status: 403,
+      message: "Only super admins can access this endpoint",
+    });
+
+  const { orgId } = req.params;
+
+  const org = await findOrganizationById(orgId);
+
+  if (!org)
+    return res.status(400).json({
+      status: 400,
+      message: "org not found",
+    });
+
+  const userIds = org.members.map((m) => m.id);
+  const users = await findUsersByIds(userIds);
+  return res.status(200).json({
+    users,
+  });
+}
+
+export async function updateUser(
+  req: AuthRequest<Partial<UserInterface>, { userId: string }>,
+  res: Response
+) {
+  if (!req.superAdmin)
+    return res.status(403).json({
+      status: 403,
+      message: "Only super admins can access this endpoint",
+    });
+
+  const { userId } = req.params;
+  const updates = req.body;
+
+  const user = await getUserById(userId);
+
+  if (!user) {
+    return res.status(400).json({
+      status: 400,
+      message: "User not found",
+    });
+  }
+
+  // If the superAdmin field is being updated, check that the user making the
+  // request is a super admin and a member of the org
+  if (Object.keys(updates).includes("superAdmin")) {
+    const context = getContextFromReq(req);
+    const memberIds = context.org.members.map((m) => m.id);
+    if (
+      !context.userId ||
+      !memberIds.length ||
+      !memberIds.includes(context.userId)
+    ) {
+      return res.status(403).json({
+        status: 403,
+        message:
+          "Only super admins that are members of an org can update its users to super admin status",
+      });
+    }
+  }
```

Comment:
This was dependent on Jeremy's PR that forces readonly permissions for non-member super admins by default, if not specified. Since we're no longer targeting that branch (he thinks its stale and should be deleted for now), I can update this code so that super admins have unrestricted access (and is consistent with how we have it today).

Reply:
Updated to allow super admins to elevate other users to super admin status, regardless of whether they're a member or not.